map/src/App.tsx

```diff
@@ -31,6 +32,7 @@ interface State {
   addInstructionsOpen: boolean;
   fullScreen: boolean;
   updateResultsOnNextClustering: boolean;
+  translate: Translate;
```

Comment:
A good way to manage this state would be to add event listeners in i18n that can update this state to filter changes down through the component tree. That would also allow things like a language changing in one window for the map also changing in all other tabs where the map is open.